Overview.src.html

```diff
@@ -2594,12 +2594,22 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <p>If <var>credentials flag</var> is set, run these substeps:
 
   <ol>
-   <li><p>Modify <var>httpRequest</var>'s
-   <span title=concept-request-header-list>header list</span>
-   per HTTP State Management Mechanism.
-   <span data-anolis-ref>COOKIES</span>
-   <!-- XXX proper hook? -->
+   <li>
+    <p>If the user agent is not configured to block cookies for <var>httpRequest</var> (see
+    <a href="https://tools.ietf.org/html/rfc6265#section-7">Section 7</a> of
+    <span data-anolis-ref>COOKIES</span>), then:
 
+    <ol>
+     <li><p>Let <var>cookies</var> be the "cookie-string" computed by passing the user agent's
+     cookie store and <var>httpRequest</var>'s <span title=concept-request-current-url>current
+     url</span> to the algorithm defined in Section 5.4 of
```

Comment:
Also, perhaps we can make that a parenthesized bit again "(see section x of y)". Below too.